public/injectEnv.js

```diff
@@ -3,7 +3,7 @@
 window.injectedEnv = {
 
   REACT_APP_APPLICATION_VERSION: 'YYYY_MM_DD/HH:MM',
-  REACT_APP_DATA_RELEASES_URL: 'https://raw.githubusercontent.com/CBIIT/CDS-Data-Releases/dev/DataReleaseNotes.json', 
+  REACT_APP_DATA_RELEASES_URL: 'https://raw.githubusercontent.com/CBIIT/CDS-Data-Releases/v11-Release/DataReleaseNotes.json', 
```

Comment:
@huuaho, Can you verify REACT_APP_DATA_RELEASES_URL's value?

Reply:
I've updated it to dev2, so we when we develop we are pointed to the current dev environment for the release notes